modules/gapi/perf/common/gapi_imgproc_perf_tests_inl.hpp

```diff
@@ -579,6 +579,91 @@ PERF_TEST_P_(SobelXYPerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+PERF_TEST_P_(LaplacianPerfTest, TestPerformance)
+{
+    compare_f cmpF;
+    MatType type = 0;
+    int kernSize = 0, dtype = 0;
+    cv::Size sz;
+    cv::GCompileArgs compile_args;
```

Comment:
Very minor comment: I'd rather use the mixedCase naming notation even if compile_args variable usually named differently as you're using it, e.g., in kernSize. But it is only my subjective opinion.

Reply:
I prefer standardization.